src/libs/Navigation/linkingConfig.js

```diff
@@ -67,6 +67,12 @@ export default {
                     Details_Root: ROUTES.DETAILS_WITH_LOGIN,
                 },
             },
+            Participants: {
+                initialRouteName: 'Participants_Root',
```

Comment:
Is there a reason for `initialRouteName` here?

Reply:
I don't think it's required as well, thanks for pointing it out.